yb-voyager/cmd/importData.go

```diff
@@ -226,6 +218,47 @@ func testYbServers(targets []*tgtdb.Target) {
 	log.Infof("all target servers are accessible")
 }
 
+func tryMatchingWithTargetHost(private_ip string, public_ip string) {
```

Comment:
You can simplify using https://pkg.go.dev/golang.org/x/exp/slices#Contains

```
func isSeedHost(names ...string) bool {
        seedHostIPList, err := lookupIP(target.Host)
        // Handle err.
        
        var allIPs []string
        for _, name := range names {
                if name != "" {
                        allIPs = append(allIPs, lookupIP(name))
                }
        }
        for _, seedHostIP := range seedHostIPs {
                if slice.Contains(allIPs, seedHostIP) {
                        return true
                }
        }
        return false;
}

func lookupIP(name string) []string {
        var result []string

        ips, err = net.LookupIP(name)
        // Handle err.
        for _, ip := range ips {
                result = append(result, ip.String())
        }
        return result
}
```

Reply:
@amit-yb that's a nice way to code it. Will do it.
Initially, i thought to `slices.Contains()` directly but was getting an error due to slice elements being net.IP instead of string.